nix/dev/formatter.nix

```diff
@@ -11,7 +11,7 @@
       treefmt = {
         flakeFormatter = true; # Enables treefmt the default formatter used by the nix fmt command
         flakeCheck = false; # Add a flake check to run treefmt, disabled, as mix format does need the dependencies fetched beforehand
-        projectRootFile = "flake.nix";
+        projectRootFile = "VERSION"; # File used to identity repo root
```

Comment:
Switched this away from `flake.nix` since there are two of those now and I didn't want that to cause any confusion for treefmt (although in my testing it seemed to be fine without this change)